contracts/src/chainlink/solidity/emergency/StarkNetValidator.sol

```diff
@@ -81,8 +96,39 @@ contract StarkNetValidator is TypeAndVersionInterface, AggregatorValidatorInterf
     payload[0] = toUInt256(status);
     payload[1] = block.timestamp;
     // Make the StarkNet x-domain call.
-    // NOTICE: we ignore the output of this call (msgHash, nonce), and we don't raise any events as the event LogMessageToL2 will be emitted from the messaging contract
-    STARKNET_CROSS_DOMAIN_MESSENGER.sendMessageToL2(L2_UPTIME_FEED_ADDR, STARK_SELECTOR_UPDATE_STATUS, payload);
+    // NOTICE: we ignore the output of this call (msgHash, nonce).
+    // We also don't raise any events as the 'LogMessageToL2' event will be emitted from the messaging contract.
+    STARKNET_CROSS_DOMAIN_MESSENGER.sendMessageToL2{value: s_fee}(
```

Comment:
Let's add a specific internal function that computes the required fee, based on the proposed workflow commented above.